src/test/java/org/fcrepo/importexport/importer/ImporterTest.java

```diff
@@ -224,6 +233,37 @@ public void testImportContainer() throws Exception {
         verify(client).put(containerURI);
     }
 
+    @Test
+    public void testLegacyModeStripsLastModified() throws Exception {
+        containerArgs.setLegacy(true);
+        final ArgumentCaptor<InputStream> streamCapture = ArgumentCaptor.forClass(InputStream.class);
+        final Importer importer = new Importer(containerArgs, clientBuilder);
+        importer.run();
+
+        verify(putBuilder).body(streamCapture.capture(), isA(String.class));
+        final Model model = createDefaultModel();
+        model.read(streamCapture.getValue(), "", "JSON-LD");
+        final Graph graph = model.getGraph();
+        System.out.println(graph);
+        assertFalse(graph.contains(Node.ANY, createURI("http://fedora.info/definitions/v4/repository#lastModified"),
+                Node.ANY));
+    }
+
+    @Test
+    public void testDefaultModeRetainsLastModified() throws Exception {
+        final ArgumentCaptor<InputStream> streamCapture = ArgumentCaptor.forClass(InputStream.class);
+        final Importer importer = new Importer(containerArgs, clientBuilder);
+        importer.run();
+
+        verify(putBuilder).body(streamCapture.capture(), isA(String.class));
+        final Model model = createDefaultModel();
+        model.read(streamCapture.getValue(), "", "JSON-LD");
+        final Graph graph = model.getGraph();
+        System.out.println(graph);
```

Comment:
Again.
I realize now that this is probably a "work in progress", so you can ignore these if you're already handling it. I'll stick them here just in case.